topic3/topic3exercise2/src/main/java/Topic3/UserArray.java

```diff
@@ -0,0 +1,64 @@
+package Topic3;
+
+import java.util.ArrayList;
+
+public class UserArray {
+	
+	private static ArrayList<User> users = new ArrayList<User>();	
+ 	private static UserArray userArray = null;
+ 
+ 	private UserArray() {	
+ 	}
+ 	
+ 	public static UserArray getInstance() {
+ 		if (userArray == null) {
+ 			userArray = new UserArray();
+ 		}
+ 		return userArray;
+ 	}
+ 	
+ 	
+ 	public void addUser(User u){
+ 		users.add(u);
+ 	}
+ 	
+ 	public void deleteUser(int id){
+ 		for (User u: users){
+ 			if (u.getId()==id){
+ 				users.remove(u);
+ 				break;
+ 			}
+ 		}
+ 	}
+ 	
+ 	public User getUser(int id){
+ 		for (User u: users){
+ 			if (u.getId()==id){
+ 				return u;
+ 			}
+ 		}
+ 		return null;
+ 	}
+ 	
+ 	public void updateUser (User user){
+ 		for (User u: users){
+ 			if (user.getId()==u.getId()){
+ 				int position=users.indexOf(u);
+ 				users.remove(u);
```

Comment:
I delete it first and then upload it again updated that is the way I think about it